src/index.html

```diff
@@ -3,15 +3,170 @@
+            name="Name" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+        
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label>
+        
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="Day_of_birth"
+          >
+        </label>
+        
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="E-mail" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="Password" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="Love-Cats?" 
+              value="yes"
+            > Yes
+            <input 
+              type="radio" 
+              name="Love-Cats?" 
+              value="no"
+            > No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color-picker"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time"
+            step="1"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your faborite brand of cars?
+          <select 
+            name="favorite cars" 
+            multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rate-work" 
+            min="0" 
+            max="100"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="field-set">
+          Comments:
+          <textarea 
+            name="comments" 
+            cols="20" 
+            rows="2" 
+            minlength="10" 
+            maxlength="100"
+          >
+          </textarea>
```

Comment:
<img width="924" alt="image" src="https://user-images.githubusercontent.com/102411130/221786095-8cedf0c7-fbad-4f02-b734-767148b1dcb9.png">
check this point

Reply:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/102411130/221786195-1c2b412e-e4b0-46d4-9277-eba1e308d0a4.png">